java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -2676,6 +2676,109 @@ public void xadd_with_options_returns_success(Pair<StreamAddOptions, String[]> o
         assertEquals(returnId, payload);
     }
 
+    @Test
+    @SneakyThrows
+    public void xtrim_with_exact_MinId() {
```

Comment:
Leave here test for `xtrim` only and move extreme options test to another place (another file)
Update `xadd` tests too

Reply:
moved to StreamCommandsTest.java

Reply:
My idea was to keep there test for `xadd` (and others) with minimal arguments or empty options AND test how non-empty options converted to command args in another test.
You moved entire test to another file, but you still combine 2 independent tests into one.